sql/hive/src/test/scala/org/apache/spark/sql/catalyst/LogicalPlanToSQLSuite.scala

```diff
@@ -101,95 +149,125 @@ class LogicalPlanToSQLSuite extends SQLBuilderTest with SQLTestUtils {
            |# Converted SQL query string:
            |$convertedSQL
            |
-           |# Original HiveQL query string:
-           |$hiveQl
+           |# Original SQL query string:
+           |$sqlString
            |
            |# Resolved query plan:
            |${df.queryExecution.analyzed.treeString}
          """.stripMargin, cause)
     }
   }
 
+  test("checkSQL") {
```

Comment:
add some comment inline explaining what we are actually testing here - also maybe rename the test name to "test behavior for the golden file test harness"